sentry/src/main/java/io/sentry/SentryTransaction.java

```diff
@@ -156,6 +158,7 @@ public void finish() {
     if (this.throwable != null) {
       hub.setSpanContext(this.throwable, this.getSpanContext());
     }
+    this.getContexts().setTrace(this.context);
```

Comment:
here `this.getContexts()` is guaranteed to not be null, right?

Reply:
Yes.